source/vir/src/sst_to_air.rs

```diff
@@ -364,6 +366,21 @@ fn assert_unsigned(exp: &Exp) {
     };
 }
 
+// Generate a unique quantifier ID and map it to the quantifier's span
+fn new_user_qid(ctx: &Ctx, exp: &Exp) -> String {
+    let fun_name = fun_as_rust_dbg(
+        &ctx.fun.as_ref().expect("Expressions are expected to be within a function").current_fun,
+    );
+    // In SMTLIB, unquoted attribute values cannot contain colons,
+    // and sise cannot handle quoting with vertical bars
+    let fun_name = str::replace(&fun_name, ":", "_");
+    let qcount = ctx.quantifier_count.get();
+    let qid = format!("{}{}_{}", air::profiler::USER_QUANT_PREFIX, fun_name, qcount);
```

Comment:
I think I'd prefer the qid structure to be defined in `def.rs`, but obviously the rest of this function needs to be here: it may be worth splitting.

Reply:
Just to clarify, are you suggesting that we move line 378 to its own function in `def.rs`?

Reply:
Yes, maybe both 376 and 378? With `fun_name` and `qcount` as arguments.
This is minor obviously.

Reply:
Okay, I moved the name generation into `def.rs`.